public/index.html

```diff
@@ -30,7 +30,7 @@
     />
     <title>Boson widgets</title>
     <script src="./scripts/zoid/zoid.js"></script>
-    <script src="./scripts/commit-button.js"></script>
+    <script src="%REACT_APP_WIDGETS_URL%/scripts/commit-button.js"></script>
```

Comment:
this line creates a loop hole, as the file is hosted at the REACT_APP_WIDGETS_URL.
Not useful in my opinion, it can be removed